src/screens/Ticketing/Purchase/Travellers/traveller-types.tsx

```diff
@@ -0,0 +1,35 @@
+import {TravellerType} from '../../../../api/fareContracts';
+
+export type Traveller = {
+  type: TravellerType;
+  text: string;
+};
+
+export type TravellerWithCount = Traveller & {count: number};
+
+export const travellerTypes: Traveller[] = [
```

Comment:
Using this temporary as the source for the traveller type options. Will use `userTypes` service or json when the model is updated to include the traveller types which are used when buying tickets.